release-notes/3.0/preview/api-diff/preview5/3.0-preview5_System.Runtime.Intrinsics.X86.md

```diff
@@ -0,0 +1,293 @@
+# System.Runtime.Intrinsics.X86
+
+``` diff
+ namespace System.Runtime.Intrinsics.X86 {
+     public abstract class Avx2 : Avx {
++        public unsafe static Vector256<short> ConvertToVector256Int16(byte* address);
```

Comment:
For reference, the overloads that take a `T*` were added since the memory load encoding reads a partial vector (rather than just a `T` or a `Vector128<T>`), so it isn't possible to implicitly emit when the user has a `LoadVector128` or `LoadScalarVector128`.